docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -98,13 +98,22 @@ The tables in each of the tabs below document configuration properties that are
     |Property|Default Value|Description|
     |---|---|---|
     |||Each service has it own collect of Clients that it uses|
-    |Protocol | http  | The protocol to use when building a URI to local the service endpoint|
-    |Host | localhost  | The host name or IP address where the service is hosted |
+    |Protocol | http | The protocol to use when building a URI to local the service endpoint|
+    |Host | localhost | The host name or IP address where the service is hosted|
     |Port | 598xx | The port exposed by the target service|
+    |UseMessageBus | false | indicate whether to use Messaging version of client |
+    |Topics |  | holds the MessageBus Topics used by the client to communicate to the service|
+    || CommandRequestTopicPrefix = edgex/core/command/request | for publishing the internal command request|
+    || CommandResponseTopic = edgex/core/command/response/# | for subscribing the internal command response|
+    || QueryRequestTopic = edgex/core/commandquery/request | for publishing the internal command query request|
+    || QueryResponseTopic = edgex/core/commandquery/response | for subscribing the internal command query response|
```

Comment:
So why does CommandResponseTopic get the wildcard in its topic, but QueryResponseTopic does not?
Is that as intended?

Reply:
This is the design in the original ADR. @lenny-intel any thought on this ?

Reply:
This is because the response is published with addition information in the topic, pre the ADR here:
https://docs.edgexfoundry.org/2.2/design/adr/0023-North-South-Messaging/#device-service-topics

Device SDK code conforms to this here: https://github.com/edgexfoundry/device-sdk-go/blob/main/internal/controller/messaging/command.go#L82

Reply:
Note that we discussed in one of the PRs to do a bit of refactoring in 3.0 on the Response topics so the subscribe is for the specific request ID. The device SDK would simply prepend the Request ID to the Response Publish topic. This would filter out responses not for the specific service.

Reply:
OK.